sass/base/_icons.scss

```diff
@@ -0,0 +1,44 @@
+.ico {
```

Comment:
.icon
these namings are not long and shortening by one letter doesn't really provide anything